cristin-person/src/test/java/no/unit/nva/cristin/person/handler/FetchFromIdentityNumberHandlerTest.java

```diff
@@ -103,9 +103,9 @@ void shouldReturnServerErrorWhenBackendAuthenticationNotSentToUpstreamOrNotValid
         handler = new FetchFromIdentityNumberHandler(apiClient, environment);
         GatewayResponse<Person> gatewayResponse = sendQuery(defaultBody(), EMPTY_MAP);
 
-        assertEquals(HttpURLConnection.HTTP_INTERNAL_ERROR, gatewayResponse.getStatusCode());
+        assertEquals(HttpURLConnection.HTTP_UNAUTHORIZED, gatewayResponse.getStatusCode());
         assertEquals(APPLICATION_PROBLEM_JSON.toString(), gatewayResponse.getHeaders().get(HttpHeaders.CONTENT_TYPE));
-        assertThat(gatewayResponse.getBody(), containsString(ERROR_MESSAGE_SERVER_ERROR));
```

Comment:
The reason this used to be server error is because we handle the authentication and not the client. So if it throws 4xx exception it is our fault by not providing an authenticated client.